### PR TITLE
HACT Value Calculations with new models

### PR DESCRIPTION
**Pull request by ntrncic**

### CODE REVIEW

File: EquiTrack/partners/serializers/partner_organization_v2.py [hunk 1]
```diff
@@ -182,8 +183,12 @@ class Meta:
             "phone_number",
             "total_ct_cp",
             "total_ct_cy",
+            "hact_values",
         )
 
+    def get_hact_values(self, obj):
```
Comment (robertavram) [inline EquiTrack/partners/serializers/partner_organization_v2.py]: remove this

File: EquiTrack/partners/models.py [hunk 2]
```diff
@@ -1495,6 +1507,8 @@ def save(self, **kwargs):
             self.total_unicef_contribution() \
             + self.partner_contribution
 
+        PartnerOrganization.planned_cash_transfers(self.intervention.agreement.partner, self)
```
Comment (robertavram) [inline EquiTrack/partners/models.py]: transactional

File: EquiTrack/partners/models.py [hunk 3]
```diff
@@ -1447,6 +1455,10 @@ class InterventionPlannedVisits(models.Model):
     spot_checks = models.IntegerField(default=0)
     audit = models.IntegerField(default=0)
 
+    def save(self, **kwargs):
```
Comment (robertavram) [inline EquiTrack/partners/models.py]: transactional

File: EquiTrack/EquiTrack/views.py [hunk 4]
```diff
@@ -206,6 +206,12 @@ class HACTDashboardView(TemplateView):
     template_name = 'hact/dashboard.html'
 
     def get_context_data(self, **kwargs):
+        q = PartnerOrganization.objects.filter(
+                Q(documents__status__in=[
```
Comment (robertavram) [inline EquiTrack/EquiTrack/views.py]: remove thsi

Comment (ntrncic) [inline EquiTrack/EquiTrack/views.py]: done

File: EquiTrack/partners/models.py (hunk 3 above)
Comment (ntrncic) [inline EquiTrack/partners/models.py]: done

File: EquiTrack/partners/models.py (hunk 2 above)
Comment (ntrncic) [inline EquiTrack/partners/models.py]: done

**Pull request merged**